### PR TITLE
Feat/5129  improve kanban

### PR DESCRIPTION
**Pull request by gbonaspetti**
#5129 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by inkhey — APPROVED** [review]
Ok for the backend part

File: frontend_app_kanban/i18next.scanner/de/translation.json [hunk 1]
```diff
@@ -1,38 +1,41 @@
...
+  "Are you sure you want to delete this column?": "Sind Sie sicher, dass Sie diese Spalte löschen möchten?",
+  "Hide": "Ausblenden",
+  "Timeline": "Zeitleiste",
+  "Tags": "Markierungen",
+  "New Kanban board": "Neue Kanban-Tafel",
+  "Board's name": "Name der Tafel",
+  "This content is deleted": "Dieser Inhalt wird gelöscht",
+  "Restore": "Wiederherstellen",
+  "Description": "Beschreibung",
+  "Value:": "__NOT_TRANSLATED__",
```
Comment (Haskaris) [inline frontend_app_kanban/i18next.scanner/de/translation.json:38]: Still need translation here

Comment (Haskaris) [inline frontend_app_kanban/i18next.scanner/de/translation.json:38]: Still need translation here

File: backend/tracim_backend/models/data.py [hunk 2]
```diff
@@ -207,7 +207,13 @@ def get_size(self, include_deleted: bool = False, include_archived: bool = False
...
                 continue
             if revision.depot_file:
-                size += revision.depot_file.file.content_length
+                try:
+                    size += revision.depot_file.file.content_length
+                except IOError:
+                    logger.warning(
+                        self, "Cannot get depot_file {}".format(revision.depot_file.file_id)
+                    )
+                    pass
```
Comment (Haskaris) [inline backend/tracim_backend/models/data.py:216]:
```suggestion
```
No need of `pass` here

**Review by knodelle — APPROVED** [review]

**Review by Haskaris — APPROVED** [review]

File: frontend_lib/src/component/Timeline/Revision.jsx [hunk 3]
```diff
@@ -15,6 +15,8 @@ const Revision = props => {
     ? revisionType.label(props.status.label)
     : props.t(revisionType.label)
 
+  console.log("REVISION")
```
Comment (inkhey) [inline frontend_lib/src/component/Timeline/Revision.jsx]: remove this.

**Review by inkhey — CHANGES_REQUESTED** [review]

File: frontend_lib/src/component/Timeline/Revision.jsx (hunk 3 above)
Comment (gbonaspetti) [reply frontend_lib/src/component/Timeline/Revision.jsx]: done

**Review by inkhey — APPROVED** [review]
ok for backend.

**Comment by knodelle** [comment]
tested on branch

**Review by inkhey — APPROVED** [review]

**Comment by gbonaspetti** [comment]
The tests frontend are canceled to allow to go faster, but they passed in the other commits and follow the modifications are just in backend.

**Pull request merged**